build/push-release-tarball.sh

```diff
@@ -5,12 +5,11 @@ set -euxo pipefail
 # This file uses `bash` and not `sh` due to the `time` builtin (the external
 # `time` is not available on CircleCI).
 
-BUCKET_NAME="binaries.cockroachdb.com"
```

Comment:
if you want to leave it configurable, it would be better to use something like:
`BUCKET_NAME="${BUCKET_NAME:-binaries.cockroachdb.com}"`